test/reuse/mobile/element/test.locator.local.android.device.conf.js

```diff
@@ -0,0 +1,349 @@
+const path = require("path");
+const merge = require("deepmerge");
+const profile = require("../../../helper/configurations/mobile.conf");
+
+exports.config = merge(profile.config, {
+    //
```

Comment:
actually would only keep the necessary configuration options and delete the comments